cli/plasmo/templates/static/react17/content-script-ui-mount.tsx

```diff
@@ -83,11 +107,41 @@ async function createShadowContainer() {
   return container
 }
 
-window.addEventListener("load", async () => {
-  const rootContainer =
-    typeof Mount.getRootContainer === "function"
-      ? await Mount.getRootContainer()
-      : await createShadowContainer()
+const createRootContainer =
+  typeof Mount.getRootContainer === "function"
+    ? Mount.getRootContainer
+    : createShadowContainer
+
+window.addEventListener("load", () => {
```

Comment:
There's a lot of moving parts here. I'd suggest splitting the logic up into different functions and calling them from the load listener rather than having the logic directly in here. 

I'd also suggest writing down comments to explain the intricacies of what's going on here.